test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +35,12 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        Object object = new Object();
+        assertFalse(Utils.isAnyNull(object));
```

Comment:
Instead of making a new object at line 40, consider this instead for brevity:
`assertFalse(Utils.isAnyNull(new Object()))`